ext/lib/gui/lvgl/lv_conf.h

```diff
@@ -0,0 +1,321 @@
+/**
+ * @file lv_conf.h
+ *
+ */
+
+#ifndef LV_CONF_H
+#define LV_CONF_H
+
+/*----------------
+ * Dynamic memory
+ *----------------*/
+
+/* Memory size which will be used by the library
+ * to store the graphical objects and other data */
+#define LV_MEM_CUSTOM      1                /*1: use custom malloc/free, 0: use the built-in lv_mem_alloc/lv_mem_free*/
+#if LV_MEM_CUSTOM == 0
+#define LV_MEM_SIZE    (32U * 1024U)        /*Size memory used by `lv_mem_alloc` in bytes (>= 2kB)*/
+#define LV_MEM_ATTR                         /*Complier prefix for big array declaration*/
+#define LV_MEM_AUTO_DEFRAG  1               /*Automatically defrag on free*/
+#else       /*LV_MEM_CUSTOM*/
+#define LV_MEM_CUSTOM_INCLUDE <kernel.h>   /*Header for the dynamic memory function*/
+#define LV_MEM_CUSTOM_ALLOC   k_malloc       /*Wrapper to malloc*/
```

Comment:
Moved from #9149
> @pizi-nordic
> What about using this library from userspace (k_malloc() is not available there)?
> What about using dedicated memory pool for graphics (in order to prevent memory exhausting by non-critical subsystem)?

Reply:
@pizi-nordic 
Added support for heap and memory pools either in kernel or user space